swift/1-Two-Sum.swift

```diff
@@ -0,0 +1,14 @@
+class Solution {
```

Comment:
Can we remove the `class Solution` as it will create duplicate classes with the same name which will result in a compilation error?